src/main/java/edu/cornell/kfs/fp/document/service/impl/CuDisbursementVoucherTaxServiceImpl.java

```diff
@@ -80,4 +86,48 @@ protected boolean validateNRATaxInformation(DisbursementVoucherDocument document
         }
         return true;
     }
+    
+	public boolean isForeignVendorWithTaxReviewRequired(String payeeTypeCode,
+			String paymentReasonCode, Integer vendorHeaderId) {
+
+		return isForeignVendor(payeeTypeCode, vendorHeaderId) && !paymentReasonDoesNotRequireTaxReviewForForeignVendor(paymentReasonCode);
+	}
+	
+	public boolean isForeignVendoAndNOTaxReviewRequired(String payeeTypeCode,
+			String paymentReasonCode, Integer vendorHeaderId) {
+
+		return isForeignVendor(payeeTypeCode, vendorHeaderId) && paymentReasonDoesNotRequireTaxReviewForForeignVendor(paymentReasonCode);
+	}
+	
+	protected boolean isForeignVendor(String payeeTypeCode, Integer vendorHeaderId){
+		if (payeeTypeCode.equals(KFSConstants.PaymentPayeeTypes.VENDOR) && vendorService.isVendorForeign(vendorHeaderId)){ 
+			return true;
+		}
+		
+		return false;
+	}
+	
+	protected boolean paymentReasonDoesNotRequireTaxReviewForForeignVendor(String paymentReasonCode){
+		if (parameterEvaluatorService.getParameterEvaluator(DisbursementVoucherDocument.class, CuDisbursementVoucherConstants.PAYMENT_REASONS_THAT_DO_NOT_REQUIRE_TAX_REVIEW_FOR_FOREIGN_VENDOR, paymentReasonCode).evaluationSucceeds()) {
```

Comment:
Ok, that makes sense. I would recommend tweaking this method, though, so that it returns the result of the condition. That way, you don't need the extra "return true/false" statements.